src/Shared/Microsoft.DotNet.Internal.DependencyInjection.Testing.Tests/Dependencies.cs

```diff
@@ -0,0 +1,131 @@
+namespace Microsoft.DotNet.Internal.DependencyInjection.Testing.Tests
```

Comment:
Glad the tests have tests so we can test that the tests test the tests correctly, :P

Reply:
Indeed.  The logic to check the scoping is weird and complicated, so I wanted a way to make sure we detected it without having to change bad code. :-) In general, testing tests is silly, but these tests are hundreds and hundreds of lines of code, so needed some simple validation. :-)